### DISCUSSION TITLE
Add a notice that current version of docs is for v3.

### DISCUSSION THREAD

**Discussion by senadir** [TanStack/query#1431] [Ideas]
I spend a good 20mn debugging why `useQueryClient` is not being exported from react-query before discovering that I installed react-query a week ago in version 2.6 and the docs are currently showing v3 examples, a small notice or dropdown (that you're viewing v3 docs, see v2 docs) would be helpful to remind people to check versions.

Thank you for the hard work, this is not a complain, but a suggestion.

**Comment by mdanics** [comment]
For anyone who came here from Google, what you're looking for is `useQueryCache`

**Comment by tannerlinsley** [comment]
We're working on a doc switcher :)
